src/psyclone/psyir/transformations/parallel_loop_trans.py

```diff
@@ -83,24 +85,30 @@ def validate(self, node, options=None):
 
         :param node: the node we are checking.
         :type node: :py:class:`psyclone.psyir.nodes.Node`
-        :param options: a dictionary with options for transformations.\
-                        This transform supports "collapse", which is the\
-                        number of nested loops to collapse.
+        :param options: a dictionary with options for transformations.
         :type options: Optional[Dict[str, Any]]
-        :param int options["collapse"]: number of nested loops to collapse
-                                        or None.
+        :param bool|int options["collapse"]: if it's a bool and is False
+            (default), it won't collapse. If it's a bool and is True, it will
+            collapse as much as possible. If it's an integer, it will attempt
+            to collapse until the specified number of loops (if they exist and
+            are safe to collapse them). The options 'ignore_dependencies_for'
```

Comment:
s/them//

Reply:
Also, does this work outermost->inwards or innermost->outwards?

Reply:
Removed them (and fixed typo)

I am unsure what you mean in the second comment. It collapses 'x' perfectly nested loops, it has no direction.

Reply:
Yes, don't know what I was getting confused about there.